### PR TITLE
add meaningful error messages for pmemaccess QEMU socket

### PR DESCRIPTION
**Pull request by Wenzel**
This patch just adds meaningful error messages when the pmemeaccess socket creation fails on the QEMU side.

Example:

~~~
$ virsh -c qemu:///system qemu-monitor-command nitro_win7x64 '{"execute": "pmemaccess", "arguments": {"path": "/etc/vmi.socket"}}'
{"id":"libvirt-20","error":{"class":"GenericError","desc":"Qemu pmemaccess: bind failed: Permission denied"}}
~~~

### CODE REVIEW

File: tools/qemu-kvm-patch/kvm-qemu-v2.8-libvmi.patch [hunk 1]
```diff
@@ -352,6 +351,12 @@ index 0000000..21c7137
 +};
 +
 +#endif /* MEMORY_ACCESS_H */
+diff --git a/pixman b/pixman
+--- a/pixman
++++ b/pixman
+@@ -1 +1 @@
+-Subproject commit 87eea99e443b389c978cf37efc52788bf03a0ee0
++Subproject commit 87eea99e443b389c978cf37efc52788bf03a0ee0-dirty
```
Comment (tklengyel) [inline tools/qemu-kvm-patch/kvm-qemu-v2.8-libvmi.patch]: This should not be part of the patch.

Comment (Wenzel) [reply tools/qemu-kvm-patch/kvm-qemu-v2.8-libvmi.patch]: Sorry for the `pixman`, again :)
